tests/e2e/gateway/02-assert.yaml

```diff
@@ -0,0 +1,212 @@
+      labels:
+        app.kubernetes.io/component: gateway
+        app.kubernetes.io/created-by: tempo-controller
+        app.kubernetes.io/instance: foo
+        app.kubernetes.io/managed-by: tempo-controller
+        app.kubernetes.io/name: tempo
+    spec:
+      containers:
+      - args:
+        - --web.listen=0.0.0.0:8080
+        - --web.internal.listen=0.0.0.0:8081
+        - --traces.write.endpoint=tempo-foo-distributor:4317
+        - --traces.read.endpoint=tempo-foo-query:16686
+        - --grpc.listen=0.0.0.0:8090
+        - --rbac.config=/etc/tempo-gateway/cm/rbac.yaml
+        - --tenants.config=/etc/tempo-gateway/secert/tenants.yaml
+        - --log.level=warn
+        image: pavolloffay/observatorium-api:001
+        livenessProbe:
+          failureThreshold: 10
+          httpGet:
+            path: /live
+            port: 8081
+            scheme: HTTP
+          periodSeconds: 30
+          successThreshold: 1
+          timeoutSeconds: 2
+        name: tempo-gateway
+        ports:
+        - containerPort: 8090
+          name: grpc-public
+          protocol: TCP
+        - containerPort: 8081
+          name: internal
+          protocol: TCP
+        - containerPort: 8080
+          name: public
+          protocol: TCP
+        readinessProbe:
+          failureThreshold: 12
+          httpGet:
+            path: /ready
+            port: 8081
+            scheme: HTTP
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 1
+        resources: {}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          readOnlyRootFilesystem: true
+        volumeMounts:
+        - mountPath: /etc/tempo-gateway/cm
+          name: rbac-rego
+          readOnly: true
+        - mountPath: /etc/tempo-gateway/secert/tenants.yaml
+          name: tenant
+          readOnly: true
+          subPath: tenants.yaml
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      serviceAccount: tempo-foo-serviceaccount
+      serviceAccountName: tempo-foo-serviceaccount
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: rbac.yaml
+            path: rbac.yaml
+          - key: tempo-gateway.rego
+            path: tempo-gateway.rego
+          name: tempo-foo-gateway
+        name: rbac-rego
+      - name: tenant
+        secret:
+          defaultMode: 420
+          items:
+          - key: tenants.yaml
+            path: tenants.yaml
+          secretName: tempo-foo-gateway
+---
+apiVersion: v1
+data:
+  tenants.yaml: dGVuYW50czoKLSBuYW1lOiB0ZXN0LW9pZGMKICBpZDogdGVzdC1vaWRjCiAgb2lkYzoKICAgIGlzc3VlclVSTDogaHR0cHM6Ly9kZXgua2xpbWxpdmUuZGUvZGV4CiAgICAKICBvcGE6CiAgICBxdWVyeTogZGF0YS50ZW1wby5hbGxvdwogICAgcGF0aHM6CiAgICAtIC9ldGMvdGVtcG8tZ2F0ZXdheS9jbS9yYmFjLnlhbWwKICAgIC0gL2V0Yy90ZW1wby1nYXRld2F5L2NtL3RlbXBvLWdhdGV3YXkucmVnbw==
```

Comment:
Do we want to check this value too?

Reply:
Ok, you want to ensure the content is that one.

Reply:
if the value is the same, the content is the same.